adapters/ddf-adapter/src/main/java/com/connexta/ion/replication/adapters/ddf/DdfNodeAdapter.java

```diff
@@ -337,15 +337,6 @@ QueryRequest createDdfQueryRequest(QueryRequest queryRequest) {
     }
     filters.add(CqlBuilder.equalTo(Constants.METACARD_TAGS, Constants.DEFAULT_TAG));
 
-    final List<String> deletedFailedItemFilters = new ArrayList<>();
```

Comment:
I removed this since we query the failed items in a separate query.

Reply:
I think this code was actually here to fetch any items that somehow failed to delete. It couldn't be bunched in with the other failed items because for those we do a query by id, which can't be grouped with any other filter because DDF will ignore it. You said removing this fixed the delete functionality though?